jump/src/context.rs

```diff
@@ -21,6 +21,22 @@ use crate::placeholders::{self, Item, Placeholder, ScieBindingEnv};
 use crate::process::{EnvVar, Process};
 use crate::{config, CurrentExe, EnvVars, Jump, Source};
 
+#[cfg(all(
+    target_os = "linux",
+    target_arch = "arm",
+    target_pointer_width = "32",
+    target_endian = "little"
+))]
+const ARCH: &str = "armv7l";
```

Comment:
This should be exported from the jump crate and re-used like EOF magic is in the package crate: https://github.com/a-scie/jump/blob/60015e988144d8b4886ca17e5bb13c02c5fcd35a/package/src/main.rs#L12-L38